### DISCUSSION TITLE
希灵帝国征兵宣传

### DISCUSSION THREAD

**Discussion by RuomoPao** [share121/inter-knot#3165] [其他]
![6bc3af3df8dcd10027816df4758b4710b8122fce](https://github.com/user-attachments/assets/3f2eb127-6764-4538-a759-c82f36428ecc)
力量即是正义！
强权带来自由！
独裁维护和平！
利刃所向即是帝国边疆！
征服一切!征服一切! 

我们生于战火 
行走虚空 
我们前进 
带来帝国征服 
皇帝在注视 
真理等待探索 
皇帝将指引 
深渊必将毁灭 
进攻，战斗，征服 
我们从不后退 
我们所向 
即是帝国锋芒 
从终点出发 
起点是到达 
我们播撒力量,追求正义 
我们坚信独裁，维护和平 
我们即是强权，带来自由或枷锁 
为了帝国 
我们汇聚于星河之下 
听主炮轰鸣 
看荣耀撒下光辉 
创造伴生毁灭 
破坏与保护同在 
远征，远征 
边疆是我们的故乡 
狂热带来信仰 
君临者永恒 
是帝国荣光 
刀锋斩断迷茫 
不懂失败，我们从不绝望 
前进，探索，征服 
胜利即是希灵 
帝国在复苏 
我们就是帝国边疆! 
                                                   ——希灵帝国征兵宣传部

**Comment by cpu644** [comment]
希灵怎么打这来了

**Comment by March-777th** [comment]
帝国？何时来的<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >来都来了顺便救救艾利都吧<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >
